activiti-spring-boot/spring-boot-starters/activiti-spring-boot-starter/src/test/java/org/activiti/starter/tests/cmdendpoint/WaitUtil.java

```diff
@@ -0,0 +1,21 @@
+package org.activiti.starter.tests.cmdendpoint;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+
+public class WaitUtil {
```

Comment:
I guess that this class could be moved inside the project `activiti-spring-boot-test` to be shared with other tests.